status-im/status-react
# Selection color changed for desktop texts and inputs

fixes #5103 

### Summary:

`:selection-color` property added in a few places

### Steps to test:
- Open Status
- check selection in chat, profile, add contact screens

status: ready
